pyphare/pyphare/pharein/__init__.py

```diff
@@ -253,7 +253,7 @@ def as_paths(rb):
 
     #### adding diagnostics
     diag_path = "simulation/diagnostics/"
-    for diag in simulation.diagnostics:
+    for _, diag in simulation.diagnostics.items():
```

Comment:
`for diag in simulation.diagnostics.values():`

?

might need `for diag in list(simulation.diagnostics.values()):`